homeassistant/components/remote/__init__.py

```diff
@@ -38,8 +41,12 @@
 
 SCAN_INTERVAL = timedelta(seconds=30)
 SERVICE_SEND_COMMAND = 'send_command'
+SERVICE_SEND_COMMANDS = 'send_commands'
 SERVICE_SYNC = 'sync'
 
+DEFAULT_NUM_REPEATS = '1'
+DEFAULT_DELAY_SECS = '0.4'
```

Comment:
In response to your first question, no this PR doesn't experience that issue. The reason that behavior occurred within the script was due to how commands were being sent in the pyharmony library. I've worked with the developer of that to get that issue fixed.

With that issue gone, 0.4 seconds was the appropriate delay for my setup in which I was using the default delay for Harmony Hubs (didn't even realize there was a setting to change for that).

And yes, I did some testing when I was testing this and 0.4s seemed about right. Any faster (0.1 or 0.2) and I would drop commands. There was no benefit to going any slower.

Take this with a grain of salt since I'm fairly new to HASS development, but I think that it would be possible to add a default delay value to the configuration.yaml that would override the 0.4 seconds.